### PR TITLE
Wf-ingress-annotations

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: manifests/runtime.yaml [hunk 1]
```diff
@@ -1,26 +1,26 @@
 apiVersion: codefresh.io/v1alpha1
 kind: Runtime
 metadata:
-  name: "{{ name }}"
-  namespace: "{{ namespace }}"
+    name: '{{ name }}'
```
Comment (ghost) [inline manifests/runtime.yaml:4]: all the white spaces here were caused by the format-on-save I recently added. I can revert it.

Comment (roi-codefresh) [reply manifests/runtime.yaml:4]: no need

**Review by roi-codefresh — APPROVED** [review]

**Pull request merged**